generator/material.sc

```diff
@@ -1307,7 +1307,7 @@ val componentCollection = WebComponentCollection(
           tag = "mwc-list-item",
           description = "List item",
           link = "https://github.com/material-components/material-components-web-components/tree/master/packages/list",
-          npmPackage = NpmPackage(name = "@material/mwc-list", version = "0.18.0"),
+          npmPackage = NpmPackage(name = "@material/mwc-list/mwc-list-item", version = "0.18.0"),
```

Comment:
Like I mentioned, this is not sufficient - this will result in correct import, but incorrect npm package. It works because you did not regenerate build.sbt.